numba/core/event.py

```diff
@@ -423,3 +426,42 @@ def on_exit(*exc_details):
 
         start_event(kind, data=data)
         yield
+
+
+def _write_chrome_trace(rec):
```

Comment:
```suggestion
def _chrome_trace_parse_records(rec):
```
suggest changing the name to something that doesn't involve writing, as it doesn't write?